src/RMP/Translate/Translate.php

```diff
@@ -90,6 +90,11 @@ public function translate($key, $substitutions = array(), $pluralisation = null,
         }
         $result = $this->_translate($key, $substitutions, $pluralisation, $domain, $this->locale);
 
+        // test if the locale is the 2 letter variant which is oftent taken from ckps_language
+        if ($result === '' && strlen($this->locale) == 2) {
+            $result = $this->_translate($key, $substitutions, $pluralisation, $domain, $this->locale . '_GB');
+        }
```

Comment:
This would make `fr` become `fr_GB`, and I'm not too sure we have any French counties here.
Should it instead be a whitelist of the ones we know are GB
```
if (in_array($this->locale, array('en','cy','gd','ga'))
```